HiP-DataStore.Model/Entity/ScoreBoard.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System.Collections.Generic;
+
+namespace PaderbornUniversity.SILab.Hip.DataStore.Model.Entity
+{
+    public class ScoreBoard : SortedSet<ScoreRecord>
```

Comment:
The namespace `DataStore.Model.Entity` should only contain types that are persisted in our Mongo database. Maybe we should move `ScoreBoard` (and `UsersScoreComparer`) out of this namespace to clarify that ScoreBoards are not actually stored in MongoDB. I think `DataStore.Core.WriteModel` is fine, so `ScoreBoard` stands right next to the type `ScoreBoardIndex` which uses it.